src/data_conn.c

```diff
@@ -292,12 +301,7 @@ handle_data_conn_error(replica_t *r)
 	MTX_UNLOCK(&spec->rq_mtx);
 
 	MTX_LOCK(&r->r_mtx);
-	if (epoll_ctl(r->epollfd, EPOLL_CTL_DEL, r->iofd, NULL) == -1) {
```

Comment:
Here, `handle_data_conn_error` was returning an error if `epoll_ctl` fails which may block management thread which will be waiting for `replica_thread`.
Since it is a cleanup path, we can ignore `epoll_ctl` return value and continue further cleanup.